packages/ui/src/FormBuilder/FormUtils.tsx

```diff
@@ -0,0 +1,133 @@
+import { Form } from 'patternfly-react';
+import * as React from 'react';
+
+export interface ISynFormControl {
```

Comment:
Wonder this probably should go in the models package eventually?  Along with whatever we call the interface for the `properties` object that we pass in.

Reply:
Ah, you know what, there's actually a `ConfigurationProperty` interface in the `models` package, we should use that one I think.

Reply:
Good point. So that would be the only place where the ui package is dependent on the models package from what I can see. @riccardo-forina that cool? When I try to add `@syndesis/models` using yarn I get errors (probably just doing it wrong). I guess I could even see the UI package getting its own set of models, but I'm curious what others think.

Reply:
Ah of course.  We probably don't want that dependency now that you mention it.

Reply:
Yes, better not to add a dependency to the models in the UI package, I kind of think of it like a spy that tells me that I'm doing something wrong.